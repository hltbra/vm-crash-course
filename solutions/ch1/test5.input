push 65
push 1
pcall 255
push 66
push 1
pcall 255
