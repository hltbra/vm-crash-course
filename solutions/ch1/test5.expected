AB[0, 0]
